deployment/mixpanel.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Postfacto, a free, open-source and self-hosted retro tool aimed at helping
+# remote teams.
+#
+# Copyright (C) 2016 - Present Pivotal Software, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+#
+# it under the terms of the GNU Affero General Public License as
+#
+# published by the Free Software Foundation, either version 3 of the
+#
+# License, or (at your option) any later version.
+#
+#
+#
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+#
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#
+# GNU Affero General Public License for more details.
+#
+#
+#
+# You should have received a copy of the GNU Affero General Public License
+#
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+set -euo pipefail
+
+if [ $# -lt 1 ]; then
+  echo 'usage: ./mixpanel.sh <script-name> ...'
+  echo 'This will report the script execution to MixPanel.'
+  echo 'Use ANALYTICS=0 (skip) or ANALYTICS=1 (send) to avoid the manual check.'
```

Comment:
Consider renaming the variable to ENABLE_ANALYTICS and the value to true/false to be in line with the enable_analytics in config.js (even if they don't affect each other right now)